examples/eliza/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Eliza Example\n",
+    "\n",
+    "This example will walkthrough the steps to set up a Eliza chatbot which generates responses based on Eliza rules. We will utilize `stave` to visualize the dialogue page that allows you to chat with the bot interactively.\n",
+    "\n",
+    "###  Introduction\n",
+    "\n",
+    "Eliza chatbot is a famous rule-based chatbot invented in 1964. The rule-based and model-less nature makes it extremely suitable for demonstration purposes. This tutorial is based on `ElizaProcessor`. For more details, refer to https://github.com/asyml/forte/blob/master/forte/processors/nlp/eliza_processor.py.\n",
+    "\n",
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pip install git+https://github.com/asyml/forte.git@master\n",
+    "pip install stave==0.0.2.dev1"
```

Comment:
These are not valid python source. If you really want to install these via pip in jupyter, you should use `% pip`. However, this is also not going to work easily. It'd be better if you simply write markdown here and ask users to install these prerequisites before running the notebook (need to refresh/reload the notebook after the installation).